app/Http/Controllers/UserAdminController.php

```diff
@@ -198,7 +198,7 @@ public function addMembership($id, Request $request)
         $member->proto_username = $alias;
         $member->save();
 
-        Mail::to($user)->queue((new MembershipStarted($user))->onQueue('high'));
+        Mail::to($user)->queue((new MembershipEndSet($user))->onQueue('high'));
```

Comment:
I don't think we want to send a MembershipEndSet email whenever a membership is created.